doc/article/en-US/router-configuration.md

```diff
@@ -326,9 +326,8 @@ The function passed to `mapUnknownRoutes()` has to return:
 
         config.title = 'Aurelia';
 
-        let handleUnknownRoutes = (instruction: NavigationInstruction): {moduleId: string} => {
```

Comment:
1(why hide a correct solution) and 3(`route` is like the **id** of the `RouteConfig`) are no go for me. Adding a route field is more acceptable for me. There is a lot that can be improved in the typings, but it is hard with the current generator.